src/Appl/Ranger.f90

```diff
@@ -21,7 +24,7 @@ module Rangerclass
       !> find the range (minimum and maxium of a single bunch) and the centroid
       !> of a single beam bunch.
       !--------------------------------------------------------------------------------------
-      subroutine singlerange(partcls,nplc,nptot,range,center)
+      subroutine singlerange_standard(partcls,nplc,nptot,range,center)
```

Comment:
The `singlerange` subroutine has not been modified apart from being renamed.
The new functionality is all in the new `singlerange_filtered` subroutine.